README.md

```diff
@@ -3,40 +3,79 @@
 
 # Sprocket GitHub Action
 
-This action uses [Sprocket](https://github.com/stjude-rust-labs/sprocket) to validate and optionally lint WDL documents.
+This action provides the functionality of the [Sprocket](https://github.com/stjude-rust-labs/sprocket) tool.
 
-## Inputs
+## `check` | `lint`
 
-### `lint`
+The `check` and `lint` subcommands perform static analysis on WDL documents. The `lint` command (or the `lint: true` option to `check`) additionally enables linting rules
+
+### Inputs
+
+#### `lint`
 
 **Optional** Whether to run linting in addition to validation. Boolean, valid choices: ["true", "false"]
 
-### `exclude-patterns`
+#### `exclude-patterns`
 
 **Optional** Comma separated list of patterns to exclude when searching for WDL files.
 
-### `deny-warnings`
+#### `deny-warnings`
 
 **Optional** If specified, Sprocket `check` will fail if any `warnings` are produced.
 
-### `deny-notes`
+#### `deny-notes`
 
 **Optional** If specified, Sprocket `check` will fail if any `notes` are produced.
 
-### `except`
+#### `except`
 
 **Optional** If specified, then the listed rules will be excepted from all `sprocket check` reports. Multiple rules can be specified as a comma-separated list, e.g. `CallInputSpacing,CommandSectionMixedIndentation`. Valid options can be found at: [analysis rules](https://github.com/stjude-rust-labs/wdl/blob/main/wdl-analysis/RULES.md) and [lint rules](https://github.com/stjude-rust-labs/wdl/blob/main/wdl-lint/RULES.md).
 
-## Example usage
+### Example usage
 
-```
+```yaml
 uses: stjude-rust-labs/sprocket-action@main
 with:
+    action: check
     lint: true
     exclude-patterns: template,test
     except: TrailingComma,ContainerValue
 ```
 
+The action `lint` can be specified and is equivalent to specifying `action: check` and `lint: true`.
+
+```yaml
+uses: stjude-rust-labs/sprocket-action@main
+with:
+    action: lint
+    exclude-patterns: template,test
+    except: TrailingComma,ContainerValue
+```
```

Comment:
Is it necessary to expose `lint`? I wonder if we'd be better off just leaving it at `check`... I'm on the fence, not sure which is best.

Reply:
To me, it's not much extra effort to support it. It's the same code block as `check`. It's just a convenience function, but since it is exposed as a subcommand of `sprocket`, I figured we should include it here too. I'm open to changing that, but that was my logic for including it.

Reply:
I'm not worried about the extra maintenance or code complexity of supporting `lint` in  addition to `check`, but rather my concern is that it complicated the README/UX by some degree...

Reply:
Have thought about it some more. I think you're right that as long as `check` and `lint` both exist as subcommands for `sprocket` we should expose them here. It makes sense to be consistent.